### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by jhdulaney**
Signed-off-by: Mairi Dulaney <jdulaney@fedoraproject.org>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,77 +10,76 @@
...
 django-appconf==1.0.2
-django-bootstrap-pagination==1.6.4
+django-bootstrap-pagination==1.7.0
 django-cors-headers==2.4.0
-django-debug-toolbar==1.9.1
-django-extensions==2.1.0
-django-filter==2.0.0
+django-debug-toolbar==1.10.1
+django-extensions==2.1.3
+django-filter==1.1.0
```
Comment (madprime) [inline requirements.txt]: Did you try running the code with these updates?

When I tried running our automated testing on this locally, I got an error similar to the one referenced by this issue, indicating the downgrade of `django-filter` is a problem: https://github.com/philipn/django-rest-framework-filters/issues/226

Comment (jhdulaney) [reply requirements.txt]: fixed, and running the tests pass.  Getting some depreciation warnings that I will take care of in the next week or so.

Comment (jhdulaney) [reply requirements.txt]: (also, playing around wiht the site manually also does not reveal any issues)

**Comment by jhdulaney** [comment]
Yes, that is groovy

**Pull request merged**